src/main/kotlin/io/github/mojira/arisa/ModuleRegistry.kt

```diff
@@ -0,0 +1,379 @@
+                        if (translatedMessage != null) config[Modules.Language.messageFormat].format(
+                            translatedMessage,
+                            defaultMessage
+                        ) else defaultMessage
+
+                    addRestrictedComment(issue, text, "helper")
+                }
+            )
+        }
+
+        register(
+            "RemoveNonStaffMeqs",
+            Modules.RemoveNonStaffMeqs,
+            RemoveNonStaffMeqsModule(config[Modules.RemoveNonStaffMeqs.removalReason])
+        ) { issue ->
+            RemoveNonStaffMeqsModule.Request(
+                issue.comments
+                    .map { c ->
+                        RemoveNonStaffMeqsModule.Comment(
+                            c.body,
+                            c.visibility?.type,
+                            c.visibility?.value,
+                            ::restrictCommentToGroup.partially1(c).partially1("staff")
+                        )
+                    }
+            )
+        }
+
+        register(
+            "RemoveTriagedMeqs",
+            Modules.RemoveTriagedMeqs,
+            RemoveTriagedMeqsModule(
+                config[Modules.RemoveTriagedMeqs.meqsTags],
+                config[Modules.RemoveTriagedMeqs.removalReason]
+            )
+        ) { issue ->
+            RemoveTriagedMeqsModule.Request(
+                issue.getCustomField(config[CustomFields.mojangPriorityField]),
+                issue.getFieldAsString(config[CustomFields.triagedTimeField]),
+                issue.comments
+                    .map { c ->
+                        RemoveTriagedMeqsModule.Comment(
+                            c.body,
+                            ::updateCommentBody.partially1(c)
+                        )
+                    }
+            )
+        }
+
+        register(
+            "ReopenAwaiting",
+            Modules.ReopenAwaiting,
+            ReopenAwaitingModule()
+        ) { issue ->
+            ReopenAwaitingModule.Request(
+                issue.resolution?.name,
+                (issue.getFieldAsString("created"))!!.toInstant(),
+                (issue.getFieldAsString("updated"))!!.toInstant(),
+                issue.comments
+                    .map { c ->
+                        ReopenAwaitingModule.Comment(
+                            c.updatedDate.toInstant().toEpochMilli(),
+                            c.createdDate.toInstant().toEpochMilli()
+                        )
+                    },
+                ::reopenIssue.partially1(issue)
+            )
+        }
+
+        register(
+            "RevokeConfirmation",
+            Modules.RevokeConfirmation,
+            RevokeConfirmationModule()
+        ) { issue ->
+            RevokeConfirmationModule.Request(
+                issue.getCustomField(config[CustomFields.confirmationField]),
+                issue.changeLog.entries
+                    .flatMap { e ->
+                        e.items
```

Comment:
It would be nice to extract this kind of methods to some mapper functions and extract it to another file

Reply:
Yeah, that would probably be a bit more organized.
I kinda like having *everything* about the module's functions in one place, but the init function is a bit cluttered like this :D

Reply:
feel free to merge by the way, thats  future work :D

Reply:
I see it as a jira thing, so we should really create a jira package under infrastrucutre and put a Calls.kt and Domain.kt